lib/Page.js

```diff
@@ -62,15 +60,25 @@ Page.prototype.prepareTemplateData = function () {
   };
 };
 
+Page.prototype.collectIncludedFiles = function (includesArray) {
+  includesArray.forEach((filePath) => {
+    this.includedFiles[filePath] = true;
+  });
+};
+
 Page.prototype.generate = function () {
+  const markbinder = new MarkBind({
+    errorHandler: logger.error,
+  });
```

Comment:
Why create a `new MarkBind` instance?

Reply:
When doing a `serve` command. The page objects created on first build are stored in an array in the site. When changes are made to a file, the same page object is used to regenerate the files.

To be safe, i create a new markbind instance to retain the same behaviour as before. (Where a new page object will be generated together with a new markbind instance).